contracts/strategies/allocation/linear-qf/qf-offchain-spec.md

```diff
@@ -0,0 +1,61 @@
+Spec: Quadratic Funding with Offchain Calculation
+---------------------------------
+
+## Overview 
+Quadratic Funding (QF) is a category of mechanisms that distribute a funding pool based upon individual donations to eligible recipients. Essentially, the amount that an individual donates to a project is considered their "vote", which is weighted according to the type of QF calculation formula being used. 
+
+At time of writing (June 2023), most prominent QF formulas are too computationally expensive to be calculated on-chain at a reasonable cost. As a step towards an eventual fully on-chain QF solution, this strategy functions a hybrid solution: 
+* votes are recorded **on-chain**
+* the distribution is calculated **off-chain**
+* the final distribution is recorded **on-chain**
+
+In order to facilitate this, the contract will emit vote events so that off-chain calculation services can pull in the relevant data. 
+
+## User flows
+Full flow is shown here: 
+![Alt text](<Strategy Evaluation - QF Vote with events.png>)
+
+### Pool Creation
+- identity owner or member creates pool via `createPool` method on `Allo.sol`
+- core QF settings are recorded on allocation strategy contract, including
+    - Application start/end and voting start/end dates
+    - On-chain application requirements 
+    - Sybil Defense settings (minimum contribution, matching cap, donor matching eligibility requirements)
```

Comment:
Would these be on-chain or part of the metadata ?
min contribution feels like on-chain but the others would be part of metadata

Reply:
I think it's a mix of on-chain and off-chain metadata. My feeling is that the settings should only be on-chain if there's logic that would be used in the contract — everything else would be metadata. 

So I think the list would be:
- On-chain: application start/end, voting start/end, require registry identity True/False
- Off-chain: minimum contribution, matching cap, donor requirements

wdyt?

Reply:
LGTM.